pkgs/development/libraries/qt-5/mkDerivation.nix

```
@@ -1,17 +1,6 @@
 { lib, debug, wrapQtAppsHook }:
 
-let inherit (lib) optional; in
-
 mkDerivation:
-
-args:
-
-let
-  args_ = {
-
-    nativeBuildInputs = (args.nativeBuildInputs or []) ++ [ wrapQtAppsHook ];
-
-  };
-in
-
-mkDerivation (args // args_)
+finalAttrs: (mkDerivation finalAttrs).overrideAttrs(super: {
+  nativeBuildInputs = (super.nativeBuildInputs or [ ]) ++ [ wrapQtAppsHook ];
+})
```

```suggestion
finalAttrs: (mkDerivation finalAttrs).overrideAttrs(previousAttrs: {
  nativeBuildInputs = (previousAttrs.nativeBuildInputs or [ ]) ++ [ wrapQtAppsHook ];
})
```

let's use the naming that's used in the docs
https://github.com/NixOS/nixpkgs/pull/119942/files#diff-c032817f6a62e23767f4c7706e35eef36e8353aa5ba5d6f9bce8c6a8cc0db01dR53

---

Fixed.